modules/apps/frontend-taglib/frontend-taglib-sample-web/src/main/java/com/liferay/frontend/taglib/sample/web/internal/display/context/SearchPaginatorDisplayContext.java

```diff
@@ -0,0 +1,85 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2024 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+package com.liferay.frontend.taglib.sample.web.internal.display.context;
+
+import com.liferay.frontend.taglib.clay.servlet.taglib.util.DropdownItem;
+import com.liferay.frontend.taglib.clay.servlet.taglib.util.DropdownItemBuilder;
+import com.liferay.frontend.taglib.sample.web.constants.SamplePortletKeys;
+import com.liferay.portal.kernel.dao.search.SearchContainer;
+import com.liferay.portal.kernel.dao.search.SearchPaginationUtil;
+import com.liferay.portal.kernel.portlet.LiferayPortletRequest;
+import com.liferay.portal.kernel.portlet.LiferayPortletResponse;
+import com.liferay.portal.kernel.portlet.PortletURLUtil;
+import com.liferay.portal.kernel.portlet.SearchOrderByUtil;
+import com.liferay.portal.kernel.util.Portal;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import javax.portlet.RenderRequest;
+import javax.portlet.RenderResponse;
+
+/**
+ * @author Daniel Sanz
+ */
+public class SearchPaginatorDisplayContext {
+
+	public SearchPaginatorDisplayContext(
+		Portal portal, RenderRequest renderRequest,
+		RenderResponse renderResponse) {
+
+		_portal = portal;
+		_renderRequest = renderRequest;
+		_renderResponse = renderResponse;
+	}
+
+	public SearchContainer<DropdownItem> getSearchContainer() {
+		LiferayPortletRequest liferayPortletRequest =
+			_portal.getLiferayPortletRequest(_renderRequest);
+		LiferayPortletResponse liferayPortletResponse =
+			_portal.getLiferayPortletResponse(_renderResponse);
+
+		SearchContainer<DropdownItem> searchContainer = new SearchContainer<>(
+			_renderRequest,
+			PortletURLUtil.getCurrent(
+				liferayPortletRequest, liferayPortletResponse),
+			null, "no-items-were-found");
+
+		searchContainer.setId("DropdownItemSearchContainer");
+		searchContainer.setOrderByCol(
+			SearchOrderByUtil.getOrderByCol(
+				_renderRequest, SamplePortletKeys.CLAY_SAMPLE, "order-by-col",
```

Comment:
`SamplePortletKeys.CLAY_SAMPLE` is no longer valid here, we need to use a new constant as this portlet is different

Reply:
Done!